File: code-recipes/java/src/test/java/easyship/RetrieveOrderRecipeTest.java
package easyship;

import util.RecipeTest;

import java.util.List;

public class RetrieveOrderRecipeTest extends RecipeTest {

    protected RetrieveOrderRecipeTest() {
        super(
                new RetrieveOrderRecipe(),
                List.of(
                    "easyship-orders-getOrder",
                    "orders-getOrderItems",
                    ""
                )
        );
    }
}

File: code-recipes/php/test/easyship/RetrieveOrderRecipeTest.php
<?php

namespace Test\easyship;

use Src\easyship\RetrieveOrderRecipe;
use Test\util\RecipeTestCase;

class RetrieveOrderRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new RetrieveOrderRecipe();
        $this->responses = [
            "easyship-orders-getOrder",
            "orders-getOrderItems",
            ""
        ];
    }
}
